### PR TITLE
remove addSubmitSubscriber

### PR DESCRIPTION
**Pull request by veolys-nanne**

### CODE REVIEW

File: WorkflowFunctionAdminBundle/Form/Type/WorkflowFunctionType.php [hunk 1]
```diff
@@ -53,7 +52,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'multiple' => true,
                 'choices' => $this->getChoices(),
             ));
-        $builder->addEventSubscriber(new AddSubmitButtonSubscriber());
+        if(array_key_exists('disabled', $options)) {
```
Comment (ngilain) [inline WorkflowFunctionAdminBundle/Form/Type/WorkflowFunctionType.php]: if (array_

File: WorkflowFunctionAdminBundle/Form/Type/WorkflowRightType.php [hunk 2]
```diff
@@ -34,7 +33,9 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             'required' => false,
         ));
 
-        $builder->addEventSubscriber(new AddSubmitButtonSubscriber());
+        if(array_key_exists('disabled', $options)) {
```
Comment (ngilain) [inline WorkflowFunctionAdminBundle/Form/Type/WorkflowRightType.php]: if (array_

**Comment by veolys-nanne** [comment]
done @itkg-ngilain

**Comment by alavieille** [comment]
:+1:

**Pull request merged**